app/src/main/kotlin/com/hedvig/app/feature/claims/ui/ClaimsUtil.kt

```diff
@@ -14,9 +14,15 @@ suspend fun startClaimsFlow(
   context: Context,
   fragmentManager: FragmentManager,
   registerForResult: ((Intent) -> Unit)? = null,
+  itemType: String? = null,
```

Comment:
Could we potentially create a class for this so it's a bit more straightforward about what it is? So the type could be itemType: ItemType? = null
```kotlin
@JvmInline
value class ItemType(val name: String)
```
But it is true that we then pass it through a bundle, I wonder how this interaction works, didn't find anything by googling. Maybe it'd need to be
```kotlin
@kotlinx.parcelize.Parcelize
@JvmInline
value class ItemType(val name: String) : Parcelable
```

Reply:
Fixing!